### ISSUE TITLE
starting wrong browser with bookmarks: chromium instead of firefox

### ISSUE THREAD

**Issue by mabod** [The-Panacea-Projects/Gnomenu#83]
My default browser is firefox. Chromium is installed in parallel. All apps use firefox as the default browser. E.g. Thunderbird and nautilus open links and html files with firefox. But gnomenu opens bookmarks with Chromium. And if I deinstall chromium gnomenu does not open bookmarks at all. How can I change that behaviour? 

PS
I am using gnomenu 23-1 on Manjaro Linux

**Comment by mabod** [comment]
This issue can be closed. The descirption is not accurate. I created a new issue : 
https://github.com/The-Panacea-Projects/Gnomenu/issues/84
